src/librustdoc/doctest.rs

```diff
@@ -35,7 +35,7 @@ use crate::lint::init_lints;
 use crate::passes::span_of_attrs;
 
 #[derive(Clone, Default)]
-crate struct TestOptions {
+pub struct TestOptions {
```

Comment:
I think my only concern is the potential for future conflicts from stuff like this, but if it's fine with your Adrian, then it's fine with me.

Reply:
Good point, opened #461 to keep track of this. @bdalrhm had linking issues when not doing this and we agreed to change it in order to have the PR online.